<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BookKeeper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/.designer/backup/images/icon.ico</normaloff>C:/Users/.designer/backup/images/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,10">
      <item>
       <widget class="QLabel" name="Welcome_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Приветствие</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4,3,2">
        <item>
         <widget class="QLabel" name="library_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Библиотека:</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="library_choose_comboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="reload_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/images/images/reload.ico);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>688</width>
           <height>602</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="Profile_menu">
    <property name="title">
     <string>Профиль</string>
    </property>
    <addaction name="Info_action"/>
    <addaction name="Exit_action"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Редактировать БД</string>
    </property>
    <addaction name="add_book_action"/>
    <addaction name="change_number_action"/>
    <addaction name="remove_book_action"/>
   </widget>
   <addaction name="Profile_menu"/>
   <addaction name="menu"/>
  </widget>
  <action name="Info_action">
   <property name="text">
    <string>Информация</string>
   </property>
  </action>
  <action name="Exit_action">
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </action>
  <action name="add_book_action">
   <property name="text">
    <string>Добавить книгу</string>
   </property>
  </action>
  <action name="remove_book_action">
   <property name="text">
    <string>Удалить книгу</string>
   </property>
  </action>
  <action name="change_number_action">
   <property name="text">
    <string>Изменить количество книг</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
